docs/layouts/shortcodes/generated/hive_catalog_configuration.html

```diff
@@ -0,0 +1,18 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>metastore.client.class</h5></td>
+            <td style="word-wrap: break-word;">"org.apache.flink.table.store.shaded.org.apache.hadoop.hive.metastore.HiveMetaStoreClient"</td>
```

Comment:
Maybe we don't need this documentation. I think this option should be inner.
It is not so good to expose a shaded interface. (`IMetaStoreClient` is shaded)